spec/lib/convenient_service/support/dependency_container/export_spec.rb

```diff
@@ -34,6 +34,32 @@
     it { is_expected.to include_module(ConvenientService::Support::Concern) }
   end
 
+  example_group "hook methods" do
+    describe "#included" do
+      context "when container is NOT Module" do
+        let(:container) { Class.new }
+        let(:include_module_result) { container.include described_class }
+
+        let(:error_message) { "#{container} is NOT a Module." }
+
+        it "raises `ConvenientService::Support::DependencyContainer::Errors::NotModule`" do
+          expect { include_module_result }
+            .to raise_error(ConvenientService::Support::DependencyContainer::Errors::NotModule)
+            .with_message(error_message)
+        end
+      end
+
+      context "when container is Module" do
+        let(:container) { Module.new }
+        let(:include_module_result) { container.include described_class }
+
+        it "does NOT raise" do
+          expect { include_module_result }.not_to raise_error
+        end
+      end
```

Comment:
Please, leave an explanation of why this test is required or remove it.

Thanks.